src/modules/battery_status/analog_battery.cpp

```diff
@@ -34,21 +30,25 @@ AnalogBattery::AnalogBattery(int index, ModuleParams *parent) :
 	snprintf(param_name, sizeof(param_name), "BAT%d_A_PER_V", index);
 	_analog_param_handles.a_per_v = param_find(param_name);
 
-	snprintf(param_name, sizeof(param_name), "BAT%d_ADC_CHANNEL", index);
-	_analog_param_handles.adc_channel = param_find(param_name);
+	snprintf(param_name, sizeof(param_name), "BAT%d_V_CHANNEL", index);
+	_analog_param_handles.v_channel = param_find(param_name);
+
+	snprintf(param_name, sizeof(param_name), "BAT%d_I_CHANNEL", index);
```

Comment:
I'd rather not add this as long as we don't need it. Even `BAT%d_V_CHANNEL` might not be that useful.

Reply:
Maybe I'll need it in the future..... To add support for current sensing........ Can we leave it there as it has no negative effect?

Reply:
There is: the more params there are the more difficult it is to setup a system.

Reply:
These two parameters can be ignored by default. Change when necessary. I have exposed extra ADC channels to connect current sensors for that current monitor can not be integrated into my board.

Reply:
Are you saying you need `BAT%d_V_CHANNEL`? None of the airframes/boards currently has `BAT%d_ADC_CHANNEL` set.
If you know you'll need them, let's keep them, otherwise we can drop them.

Reply:
V_CHANNEL is required by some boards, for sure. I'm asking to add I_CHANNEL, which is required by me...

Reply:
bbblue requires `BAT_ADC_CHANNEL`, which is changed to `BATi_V_CHANNEL`